MonkeyChat/ConversationListViewController.swift

```diff
@@ -135,10 +135,18 @@ class ConversationsListViewController: UITableViewController {
     //register listener for acknowledges of opens I do
     NotificationCenter.default.addObserver(self, selector: #selector(self.openResponseReceived(_:)), name: NSNotification.Name.MonkeyConversationStatus, object: nil)
     
+    //register listener for messages updated in last conversation open
+    NotificationCenter.default.addObserver(self, selector: #selector(ConversationsListViewController.updateConversationList), name: NSNotification.Name(rawValue: MonkeyChatNotification.messageSent), object: nil)
```

Comment:
With the proposed change the struct `MonkeyChatNotification` this would be something like:
`NotificationCenter.default.addObserver(self, selector: #selector(self.updateConversationList), name: NSNotification.Name.MonkeyChatNotification.messageSent, object: nil)`